elements/pl-order-blocks/pl-order-blocks.css

```diff
@@ -14,8 +14,18 @@ div.pl-order-blocks-bottom {
 
 li.pl-order-blocks-code {
   list-style-type: none;
+  padding: 0px !important;
 }
 
+li.pl-order-blocks-code .badge{
+    margin: 5px !important;
+}
```

Comment:
Seems odd to need to do this. How are these badges being styled by the other elements? Is there some default badge style somewhere we can just refer to?

Reply:
When the format is changed to `code` for some reason the formatting of the badge gets strange. That line has since been changed to the following, just to cut down on whitespace:
```css
margin-left: 5px !important;
margin-right: 0px !important;
```
If those are excluded we get badges that look like this when the formatting is included as `code`:
![image](https://user-images.githubusercontent.com/16738261/188521545-19e8821c-95c2-44d1-8c45-2c1b9156b304.png)

Reply:
When they're added back it looks like this:
![image](https://user-images.githubusercontent.com/16738261/188521644-caa35382-d1c6-412b-8daa-64549b6ebb75.png)

Reply:
so odd. Any ideas why other elements dont have this problem?

Reply:
Not sure. :( I'm afraid I don't have enough experience with the other elements to say. Is there an example you had in mind of where a badge is placed inline with a block of pl-code I could take a look at?

Reply:
most of them do it, just start with looking at a simple one like pl-integer-input and see what you learn